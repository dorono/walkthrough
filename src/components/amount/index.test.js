import React from 'react';
import {shallow} from 'enzyme';
import Amount from './index';

const defaultProps = {
    unit: 'EC',
    children: 20000000000,
};

describe('Amount', () => {
    it('should render without errors', () => {
        shallow(<Amount {...defaultProps} />);
    });

    it('should match snapshot', () => {
        expect(shallow(<Amount {...defaultProps} />)).toMatchSnapshot();
    });
});
