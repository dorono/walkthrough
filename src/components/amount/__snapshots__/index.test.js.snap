// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Amount should match snapshot 1`] = `
<div
  className="root"
>
  <Monospaced>
    20,000,000,000 EC
  </Monospaced>
</div>
`;
